README.md

```diff
@@ -53,9 +53,31 @@ Combining `editor.suggest.preview` with the [pre-fill required fields](#code-com
 
 Completing the snippet allows you to tab complete through each attribute and block.
 
-### Syntax Validation
+### Validation
+
+While editing, the configuration is validated and invalid code is highlighted. 
+
+HCL syntax is checked for e.g. missing `}`, `"` or other "control characters" being out of place.
+
+![](docs/validation-rule-hcl.png)
+
+Enhanced validation of selected Terraform language constructs in both `*.tf` and `*.tfvars` files based on detected Terraform version and provider versions is also available from `v2.28.0`. This can highlight deprecations, missing required attributes or blocks, references to undeclared variables and more, [as documented](https://github.com/hashicorp/terraform-ls/blob/main/docs/validation.md#enhanced-validation).
```

Comment:
```suggestion
Enhanced validation of selected Terraform language constructs in both `*.tf` and `*.tfvars` files based on detected Terraform version and provider versions is also provided. This can highlight deprecations, missing required attributes or blocks, references to undeclared variables and more [as documented in](https://github.com/hashicorp/terraform-ls/blob/main/docs/validation.md#enhanced-validation).
```

Do we need to state version here? We don't start this section by gating version.

Reply:
I wouldn't add the `in` near the end though, because the link isn't standalone.